app/controllers/concerns/history_filter.rb

```diff
@@ -0,0 +1,56 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HistoryFilter
+  extend ActiveSupport::Concern
+
+  # NOTE: path_for_clear_view_filter must be defined in the including controller
+  # and added as a helper method.  Usually it's just the index path for the controller.
+  # Additionally, you'll need to add apply_view_filters(reports) to limit the reports shown.
+
+  included do
+    before_action :set_view_filter, only: [:index]
+
+    private def view_filter_params
+      params.permit(
+        :creator,
+        :start,
+        :end,
```

Comment:
Maybe add `history_filter` to these names, seem like they might conflict w/ other params for controllers this might get mixed into?

Reply:
Actually, I think that applies to all the names...

Reply:
Hmm, maybe, it shouldn't since they are bare params, but I'll see what would be involved in making that change.  Thanks.